aether-common-library/aether/common/conf/settings.py

```diff
@@ -39,7 +39,8 @@
 USE_L10N = True
 USE_TZ = True
 
-STATIC_URL = f'{APP_URL}static/'
+STATIC_URL_PREFIX = os.environ.get('STATIC_URL_PREFIX', '/')
+STATIC_URL = f'{STATIC_URL_PREFIX}static/'
```

Comment:
Sorry, what I was going to say is that the expected variable for this is `STATIC_URL`

```python
# settings.py
STATIC_URL = os.environ.get('STATIC_URL', '/static')
```

```bash
# uwsgi/start.sh
if [ ! -z "${CUSTOM_UWSGI_SERVE_STATIC:-}" ]; then
    export UWSGI_STATIC_EXPIRES=${UWSGI_STATIC_EXPIRES:-"/* 7776000"}

    STATIC_URL=${STATIC_URL:-/static}
    STATIC_DIR="/var/www/static"

    MAP_STATIC="--static-map ${STATIC_URL}=${STATIC_DIR}"
    # favicon.ico is always in root
    MAP_FAVICO="--static-map2 /favicon.ico=${STATIC_DIR}/aether/images/aether.ico"
    STATIC_CONTENT="$MAP_STATIC $MAP_FAVICO"
fi
```

Reply:
Changed to `STATIC_URL` and removed the default static append from uwsgi.